ci/test_inner.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+# -x Exit immediately if any command fails
+# -e Echo all commands being executed.
+# -u fail for undefined variables
+set -xeu
+echo "Run tests in docker"
+php ./run-tests.php -P -q --show-diff
```

Comment:
Does this need REPORT_EXIT_STATUS as well?

Reply:
Oh. I added that to the wrong command